sys-process/runit/runit-2.1.2-r6.ebuild

```diff
@@ -54,7 +54,11 @@ src_install() {
 	dobin $(<../package/commands)
 	dodir /sbin
 	mv "${ED}"/bin/{runit-init,runit,utmpset} "${ED}"/sbin/ || die "dosbin"
-	dosym ../etc/runit/2 /sbin/runsvdir-start
+	if use split-usr ; then
+		dosym ../etc/runit/2 /sbin/runsvdir-start
+	else
+		dosym ../../etc/runit/2 /sbin/runsvdir-start
+	fi
```

Comment:
A possible alternative would be to use an absolute symlink: `dosym "${EPREFIX}/etc/runit/2 /sbin/runsvdir-start`

Up to you as to which you prefer.

Reply:
after check the other ebuilds, found large part of dosym's usage is to create relative link or link in same directory, so let's keep the relative symlink.

Reply:
I don't think that the "large part of dosym's usage" matters here. You could as well say the `ln` command has as its "large part of its usage" to be "create hardlinks" therefore best to use hardlinks?

The difference between absolute and relative symlinks here is that you can avoid adding a USE flag and an if/else conditional. But in some cases absolute symlinks can be actively bad because they don't behave well at runtime (one example could be, if the symlink is for a C static/shared library, and the symlink messes with sysroot handling).

As floppym said -- whichever one you prefer, I guess. But it seemed to me like you weren't sure of the pros and cons, so I figured I'd mention some context.

Reply:
just see the post today and thanks for the comment, :-(
yes, I know the pros and cons of absolute/relative link, and I'm using chroot, so I choose relative link finally.
There are some issues being discussed about using runit as init, so I'll take this into account together when submit the final solution. I can choose hard link if we're sure /etc/runit/2 won't be changed by users after install, but it's not for sure yet.